lib/pbench/server/database/models/server_settings.py

```diff
@@ -1,3 +1,4 @@
+"""A framework to manage Pbench configuration settings."""
```

Comment:
I thought it might be a good place to bridge "configuration" and "settings."  I would like to see us use module doc strings, but if it is a real problem, I can drop it.

Reply:
I have no problem with docstrings in general (I mean, in general, I'm very much in favor of code comments of any kind), however, we need to recognize that docstrings (and all documentation) come with a cost, and the text that we include in the code needs to provide sufficient value to offset those costs.

I am also a fan of self-documenting code.  So, I would amplify Dave's second question:  does the docstring actually add value above and beyond the name of the module?  If not, it's just taking up space for no real purpose, and it should be omitted.

Reply:
For what it's worth, the new module docstring is useful and good, although I'm still on the fence about the overall value of such a convention...

Reply:
If the new docstring is useful and good, I say we keep it.  Perhaps it will inspire others of its ilk, and, even if not, it will still be useful and good where it is.